data/merfish_aging_mouse_brain/merfish_aging_mouse_brain.py

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env python
+
+# Made by Paul Kiessling pakiessling@ukaachen.de
+
+
+import argparse
+import json
+import os
+import tempfile
+import urllib.request
+
+import anndata
+import pandas as pd
+import scipy
+
+LINKS = {
+    "aging": "https://datasets.cellxgene.cziscience.com/1892a5fe-c5f4-4fa7-9dac-54216566bdbf.h5ad",
+    "inflammation": "https://datasets.cellxgene.cziscience.com/2bac088e-cc19-461f-a3dd-83466c44a122.h5ad",
```

Comment:
It sounds a bit like these are different biological conditions.

In how far are these two samples/experiments related?
I am wondering if they should be split into two separate datasets?

Reply:
They compared "similarities and notable differences in cell-activation patterns induced by aging and systemic inflammatory", but the annotation of doamins is of mouse brain areas layers and not related to the condition.

I dont think it matters for our purpose. I could however note it in the json file

Reply:
This would only matter if we ever try to use multi-sample methods, then I guess it depends on how different these biological conditions are. Maybe leave a comment in one of the metadata tracking files.